shell/platform/fuchsia/flutter/platform_view.h

```diff
@@ -41,36 +34,32 @@ using OnCreateSurface = fit::function<std::unique_ptr<flutter::Surface>()>;
 // messages are notably responsible for communication related to input and
 // external views / windowing.
 //
-// The PlatformView implements SessionListener and gets Session events but it
-// does *not* actually own the Session itself; that is owned by the
-// FuchsiaExternalViewEmbedder on the raster thread.
+// The Fuchsia PlatformView implements Scenic's SessionListener interface and
+// thus receives Session events.  However it does *not* actually own the Scenic
+// Session itself; that is owned by the FuchsiaExternalViewEmbedder which runs
+// on the raster thread.
```

Comment:
Probably a little table would help make the connections clear:

// raster thread - FuchsiaExternalViewEmbedder - fuchsia.ui.scenic.Session client - send commands
// platform thread - flutter::PlatformView  - fuchsia.ui.scenic.SessionListener server - receive events

Reply:
Obsolete